pyvocz/templates/_base.html

```diff
@@ -10,6 +10,20 @@
             <link rel="stylesheet" href="{{ url_for('static', filename='third-party/bootstrap.min.css') }}">
             <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
         {% endblock styles %}
+        <!-- For Facebook and Twitter: -->
+        <meta property="og:type" content="article">
+        <meta property="og:title" content="{{ self.title()|trim }}">
+        {%- if g.lang_code == 'cs' %}
+        <meta property="og:description" content="Pyva jsou setkání příznivců programovacího jazyka Python a spřízněných technologií.">
+        {% else %}
+        <meta property="og:description" content="Pyvo is a meetups for fans of the Python programming language and related technologies.">
+        {% endif -%}
+        <meta property="og:url" content="{{ request.url }}">
```

Comment:
Will this be full (external) URL or just the path?

Reply:
Full.